lib/v2/liveuamap/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    liveuamap: {
```

Comment:
The top level object key should be [the domain name](https://docs.rsshub.app/joinus/new-radar#top-level-object-key).